packages/smooth_ui_library/lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -44,7 +44,7 @@ class SmoothAlertDialog extends StatelessWidget {
           child: Row(
             crossAxisAlignment: CrossAxisAlignment.start,
             mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-            children: actions,
+            children: actions!,
```

Comment:
To be fixed: if `actions` is `null` (which is possible because of `?`) that will crash.
When `actions` is `null`, don't display that `Row`.